core/src/main/java/com/kin/ecosystem/core/bi/events/MaxWalletsExceededError.java

```diff
@@ -0,0 +1,241 @@
+
+package com.kin.ecosystem.core.bi.events;
+
+// Augmented by script
+import com.kin.ecosystem.core.bi.Event;
+import com.kin.ecosystem.core.bi.EventsStore;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+
+/**
+ * when a user exceeded max wallet limit
+ * 
+ */
+public class MaxWalletsExceededError implements Event {
+    public static final String EVENT_NAME = "max_wallets_exceeded_error";
```

Comment:
@neritBA Did you define this new metric?

Reply:
yes she did